src/App.js

```diff
@@ -1,10 +1,26 @@
+/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
-import CalculatorUI from './Components/calculatorUI';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+// eslint-disable-next-line import/extensions
+import CalculatorUI from './Components/CalculatorUI.js';
```

Comment:
- [ ] Your app shows an error because of the import at line 10. `CalculatorUI.js` should not be capitalized.

Reply:
done